dashboard/app/views/layouts/_footer.html.haml

```diff
@@ -8,7 +8,7 @@
         .dim!= piped_spaces
         = link_to t('footer.translate'), CDO.code_org_url('/translate')
         .dim!= piped_spaces
-        = link_to t('footer.support'), t('footer.support_url')
+        = link_to t('footer.support'), 'https://support.code.org'
```

Comment:
I forgot one more thing, can you make this one also pop up in a new tab to be consistent with the rest?